app/Providers/AppServiceProvider.php

```diff
@@ -17,19 +16,23 @@
 
 class AppServiceProvider extends ServiceProvider
 {
+    /**
+     * Register any application services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        //
+    }
+
     /**
      * Bootstrap any application services.
      *
      * @return void
      */
     public function boot()
     {
-        $this->app->singleton(GateContract::class, function ($app) {
-            return new CustomGate($app, function () use ($app) {
-                return call_user_func($app['auth']->userResolver());
-            });
-        });
```

Comment:
Related to [this comment](https://github.com/DoSomething/rogue/pull/1071/files#r459600449), since Laravel now allows guest users we no longer need the `CustomGate` class.